src/components/Modals/ManageAccounts/ManageAccountsModal.tsx

```diff
@@ -157,14 +157,21 @@ export const ManageAccountsModal = () => {
                 onClick={handleClickAddChain}
                 width='full'
                 size='lg'
+                isLoading={isDrawerOpen}
```

Comment:
Probably a @reallybeard one but I'm not sure it makes sense to consider this a loading state UX-wise? Nothing is loading per se.
Feels like having proper z-index (which is already the case) is all we need?

Reply:
Disabling is almost mandatory as the feature is disabled while the user select the accounts but at the same time disabling and the loading state looks pretty nitpick as anyway it should be hidden behind the drawer overlay and shouldn't be clickable without closing it anyway